src/serlo-editor/plugins/text/hooks/use-text-config.tsx

```diff
@@ -10,6 +10,7 @@ const defaultFormattingOptions: TextEditorFormattingOption[] = [
   TextEditorFormattingOption.math,
   TextEditorFormattingOption.richTextBold,
   TextEditorFormattingOption.richTextItalic,
+  TextEditorFormattingOption.gap,
```

Comment:
Should this actually be in the `defaultFormattingOptions`?

Reply:
No, that should not stay in the defaults. Probably there, because it was easier to do it that way initially.

Reply:
Simplified the configuration